ocs_ci/utility/bootstrap.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import os
+
+import boto3
+from libcloud.compute.providers import get_driver
+from libcloud.compute.types import Provider
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import NodeNotFoundError, UnsupportedPlatformError
+from ocs_ci.utility import vsphere
+from ocs_ci.utility.utils import run_cmd, get_infra_id
+
+logger = logging.getLogger(__name__)
+
+
+def gather_bootstrap():
+    """
+    Gather debugging data for a failing-to-bootstrap control plane.
+    Data is placed in the `gather_bootstrap` directory under the log directory.
+
```

Comment:
Add raises section please and document: NodeNotFoundError + CommandFailed I think is also very possible to see here.

Reply:
Do we include `CommandFailed` in other docstrings where that is not explicitly raised, but rather can be raised by subsequently called functions? I think our docstring should only reflect what this function raises explicitly, no?

Reply:
Done

Reply:
yep, it would be better to stick just with raised exception in the documented function